cli/.jvm/src/main/scala/aqua/AppOps.scala

```diff
@@ -6,16 +6,16 @@ import aqua.parser.expr.ConstantExpr
 import aqua.parser.lift.LiftParser
 import cats.data.Validated.{Invalid, Valid}
 import cats.data.{NonEmptyList, Validated, ValidatedNel}
-import cats.effect.ExitCode
+import cats.effect.unsafe.implicits.global
```

Comment:
Use givens instead: ExecutionContext is provided with IOApp, no need to import it here